src/components/Table/Table.module.scss

```diff
@@ -16,16 +15,17 @@
 
   .table {
     cursor: default;
+    width: 100%;
     border-collapse: collapse;
     white-space: normal;
-    color: var(--color-font-base) !important;
+    color: var(--table-color) !important;
```

Comment:
Are we able to lose this !important here? I realize it was there before your PR but would be nice to get rid of it if we can.